### ISSUE TITLE
Unpacking NamedTuples gives Any type

### ISSUE THREAD

**Issue by yellowjello** [microsoft/pyright#907]
**Is your feature request related to a problem? Please describe.**
Currently unpacking NamedTuples results in Any type for the unpacked elements.

Example:
```
from typing import NamedTuple

class Something(NamedTuple):
    blah: str
    hah: int

ntpl = Something("yes", 1)

(a, b) = ntpl # a and b are Any
```

**Describe the solution you'd like**
It would be great if the behavior is the same as unpacking regular tuples where it gives the types of the unpacked elements.

**Comment by erictraut** [comment]
This is related to #860.

**Comment by erictraut** [comment]
This is now addressed in version 1.1.63 of Pyright, which I just published. It will also be in the next release of Pylance.
